### DISCUSSION TITLE
How to include manual status in automations

### DISCUSSION THREAD

**Discussion by LKSkywalker85** [basnijholt/adaptive-lighting#921] [Q&A]
Hi all, I'm a really happy user of the AL integration! Great job!!! However, I need a way to change the behavior of an automation based on the fact if a certain lamp is in manual or not-manual way. Easy explained: I have a lamp that is triggered by multiple automations based on an FP2-sensor. If I change the lamp manually and move around one of the automations will switch off the light, the next turn it on again. As expected, it is then back in AL-mode. What I want to achieve is that those triggers are only made (turne lamp on/off) if they are in AL-mode but not if I changed the settings manually. Would be great if you could assist me. So far I have all boxes checked for manual control but I have no clue how to read-out the AL-condition for one lamp within an automation. If triggered by motion in zone 1 then turn off light, but only if AL-status is not manual.So far I'm only using UI interface, no yaml-file.

**Comment by patienttruth** [comment]
I got you as I just figured this out.

Call the service "adaptive lighting... .... Set manual control" then select the switch as either a specific part of the AL switching (i.e. brightness in my case) or set to apply to specific light entities.

I am turning on manual control right now, which disabled AL brightness control in my case.

![Screenshot_20240223-072530~2](https://github.com/basnijholt/adaptive-lighting/assets/105557996/e40d7c3a-de96-4490-998d-e2a86ac45bb3)
